### PR TITLE
Pull Request for Review of Intuit Sign Up Page Clone Before Final Submission

### PR DESCRIPTION
**Pull request by Calvinoea**
The pull request is being made for your review of the code of the Intuit Sign Up Page clone before final submission. The code has been made to translate the front-end design of the Intuit Sign Up Page, using HTML and CSS as part of the HTML Form Microverse task.

It has been depoyed on:  https://frosty-nightingale-713a3c.netlify.com/

The code is made to ensure that design principles are followed as well as HTML and CSS principles. It forms the basis of a static web page. 

HTML validation has been carried out on the html code using a HTML validator. 

The files have been served using Raw.Githack:

https://raw.githack.com/nellencr/Form-for-Collecting-Data/tree/form_section
https://rawcdn.githack.com/nellencr/Form-for-Collecting-Data/tree/form_section

### CODE REVIEW

**Review by fatymahmed — COMMENTED** [review]
## Hi @Calvinoea @nellencr,
Your project looks good 👏👏, However, there are a couple of things to fix.
- I noticed that the Pull Request is merged, I hope that this was done by mistake because you shouldn't merge a pull request unless it has been approved.
- Please add stickler to your repository as mentioned in [Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png), use [this](https://github.com/microverseinc/linters-config/tree/master/css) guide on how to do it.
- You should delete commented code as mentioned in [Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)
- Change text in your sign-in button to be similar to the one on mint page.
screenshot

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+.createaccount i {
+  font-size: 60px;
+  font-weight: 400;
+  color:white;
+  font-family: AvenirNext,Avenir,Helvetica Neue,Helvetica,Arial,sans-serif;
+cursor: pointer;
+
+font-size: 12px;
+}
+*/
```
Comment (fatymahmed) [inline styles.css:304]: - Remove the commented code.

**Review by fatymahmed — COMMENTED** [review]
## Kindly make the suggested changes then request for another code review.
- Please open a new Pull request since this one has been merged.
- Don't forget to add the link of this pull request to your second pull request

## Status: Changes Required
_Feel free to reach out to me on slack (Fatima) If you have any questions, I'd be happy to help_

**Comment by Calvinoea** [comment]
Thank you for your feedback @fatymahmed  It is highly appreciated.

**Pull request merged**